docs/walkthrough_annotate.md

```diff
@@ -1,15 +1,90 @@
 ---
 title: Annotate with eScriptorium
-summary: About annotation features in eScriptorium.
+summary: About the annotation features in eScriptorium.
 authors:
     - Alix Chagué
-date: 2023-01-31
+date: 2023-02-22
 ---
 
-# Walkthrough : annotation with eScriptorium
+# Walkthrough: annotation with eScriptorium
 
-## Region and line types
+???+ Warning
+    This section details the "Annotation" components, not the "Segmentation" features, which are two different things.
 
-## Image annotation
+Annotation can be performed at two levels:  
 
-## Text annotation
+- at image level
+- at text level
+
+The different catagories that you want to use for annotation are activated and managed from a document's "Ontology" page (also accessible at `{base_url}/document/{doc_id}/ontology`). See in particular in the 3 last items of the page:  
+
+- Annotation components
+- Image Annotations
+- Text Annotations
+
+![image: screenshot of the section of Ontology dedicated to image and text annotations](img/annotate/ontology.png "The three items in Ontology dedicated to image and text annotation.")
+
+## Annotation components
+
+!!! Warning
+    This section needs to be completed.
+
+## Image annotations
+
+### Create a new category
+
+To create an new category for image annotation, go to the "Ontology" tab, then in "Image Annotations", fill a new line with the following information:  
+
+- Name of the category
+- Abbreviation (3 letters max)
+- Type
+- Select the marker type (rectangle of polygon) depending on the type of shape you want to draw
+- Leave the default color or change it
+- Check "Allow Comments" if you want to be able to add comment to the image annotations
```

Comment:
"add commentS" --> I think it should have a "s" and if not, it should be "add a comment"